darts/utils/data/simple_inference_dataset.py

```diff
@@ -11,17 +11,88 @@
 
 
 class SimpleInferenceDataset(TimeSeriesInferenceDataset):
+
     def __init__(self,
                  series: Union[TimeSeries, Sequence[TimeSeries]],
-                 covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None):
+                 covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+                 n: int = 1,
+                 input_chunk_length: int = 12,
+                 output_chunk_length: int = 1):
+        """
+        Creates a dataset from lists of target series and corresponding covariate series and emits
+        3-tuples of (tgt_past, cov_past, cov_future), all `TimeSeries` instances.
+        `tgt_past` corresponds to the target series, which will be predicted into the future.
+        `cov_past` is equal to the covariates with the same time index as the target series if covariates
+        are provided, otherwise a value of `None` will be emitted.
+        Both `tgt_past` and `cov_past` will be `input_chunk_length` long.
+        If the model is required to produce the forecast over multiple iterations, i.e. if
+        `n > self.output_chunk_length`, and if covariates were provided, then `cov_future` will
+        be equal to the future covariates up to `n - self.output_chunk_length` time steps into the future.
+
+        Parameters
+        ----------
+        series
+            The target series that are to be predicted into the future.
+        covariates
+            Optionally, the corresponding covariates that are used for predictions. This argument is required
+            if the model was trained with covariates.
+        n
```

Comment:
Perhaps a couple of comment lines to explain why these three parameters are needed would be nice (for our future selves ;)